runcombine.py

```diff
@@ -73,8 +73,9 @@ def call_combine(cmd):
         " -M AsymptoticLimits --datacard cards-{name}/combined.root"
         #" -M FitDiagnostics -datacard cards-{name}/combined.root --plots signalPdfNames='ADD*,Signal' --backgroundPdfNames='*DY*,*WW*,*TOP*,ZZ*,WZ*,VVV'"
         " -m 125 --cl 0.95 --name {name} {options}"
-        ##" --rMin=0 --cminFallbackAlgo Minuit2,Migrad,0:0.05"
-        " --X-rtd MINIMIZER_analytic --X-rtd FAST_VERTICAL_MORPH".format(
+        " --rMin=0 --cminFallbackAlgo Minuit2,Migrad,0:0.05"
+        " --X-rtd MINIMIZER_analytic --X-rtd FAST_VERTICAL_MORPH --run blind".format(
+        #" --X-rtd MINIMIZER_analytic --X-rtd FAST_VERTICAL_MORPH".format(
```

Comment:
I was just wondering if the combine arguments '--X-rtd MINIMIZER_analytic' , '--X-rtd FAST_VERTICAL_MORPH' and  '--cminFallbackAlgo Minuit2,Migrad,0:0.05' are needed?  The combine arguments -m 125 --cl 0.95 can maybe be removed because we don't use the $m in our datacards and the 0.95 cl is default.